demisto_sdk/dev_tools/lint_manager.py

```diff
@@ -251,7 +251,7 @@ def _print_final_results(self, good_pkgs: List[str], fail_pkgs: List[str]) -> in
         Returns:
             int. 0 on success and 1 if any package failed
         """
-        if self.circle:
+        if self.outfile:
             self.create_failed_unittests_file(fail_pkgs)
```

Comment:
```suggestion
            self.create_failed_unittests_file(fail_pkgs, self.outfile)
```

Reply:
pass outfile to create_failed_unittests_file and make create_failed_unittests_file to be staticfile